README.md

```diff
@@ -31,12 +31,12 @@ Polaris Catalog is open source under an Apache 2.0 license.
 
 ## API Docs
 
-API docs are hosted via Github Pages at https://polaris-catalog.github.io/polaris. All updates to the main branch
+API docs are hosted via Github Pages at https://polaris.io. All updates to the main branch
 update the hosted docs.
 
-The Polaris management API docs are found [here](https://polaris-catalog.github.io/polaris/index.html#tag/polaris-management-service_other)
+The Polaris management API docs are found [here](https://polaris.io/index.html#tag/polaris-management-service_other)
 
-The open source Iceberg REST API docs are found [here](https://polaris-catalog.github.io/polaris/index.html#tag/Configuration-API)
+The open source Iceberg REST API docs are found [here](https://polaris.io/index.html#tag/Configuration-API)
```

Comment:
Minor: not necessarily related to this PR, but we may call it "Apache Iceberg" or "Iceberg" instead of "open source Iceberg"

Reply:
Good catch! Updating :)